tools/cxbe/Xbe.cpp

```diff
@@ -1102,8 +1102,6 @@ void Xbe::DumpInformation(FILE *x_file)
 
     time_t dwTimeDate = m_Header.dwTimeDate;
 
-    fprintf(x_file, "XBE information generated by CXBX (Version " _CXBX_VERSION ")\n");
```

Comment:
We should probably keep this message, but modify it to get rid of version number; ideally, add git hash instead.